pkg/storage/replicate_queue.go

```
@@ -305,6 +314,12 @@ func (rq *replicateQueue) processOneChange(
 			rangeInfo,
 		)
 		if err != nil {
+			// If there are more replicas than addToPurgatoryReplicaCountThreshold,
+			// don't bother sending to purgatory.
+			if _, ok := err.(*allocatorError); ok && rq.store.ReplicaCount() > addToPurgatoryReplicaCountThreshold {
+				log.VErrEvent(ctx, 1, err.Error())
```

Give this some context so that we know why these errors are being dropped.